doctr/travis.py

```diff
@@ -401,3 +389,22 @@ def push_docs(deploy_branch='gh-pages'):
     run(['git', 'pull', 'doctr_remote', deploy_branch])
     print("Pushing commit")
     run(['git', 'push', '-q', 'doctr_remote', deploy_branch])
+
+def determine_push_rights(branch_whitelist):
```

Comment:
Should we test that `branch_whitelist` is a container (or at least not a string?)